### ISSUE TITLE
Request-Block Has Sky Access?

### ISSUE THREAD

**Issue by Sixdd6** [cubex2/customstuff4#37]
Basically the title, check if a block has access to the sky (no blocks above it).

**Comment by Sixdd6** [comment]
Basically I've got an idea for a bee mutating mod and I would want to have a basic check to make sure that the "apiary" is "outside".

I'd actually like to add a request for checking if the block is "nearby" to other blocks as well, for example to check for flowers around the main block for bee honey production, or possibly resource production with ore blocks nearby.

**Comment by cubex2** [comment]
So you want to have an additional requirement for machines to run?

**Comment by Sixdd6** [comment]
Yes, two if they are both possible. Check for blocks above and check for blocks in radius.
